api/TrustedHTML.json

```diff
@@ -46,8 +48,60 @@
           "deprecated": false
         }
       },
+      "toJSON": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TrustedHTML/toJSON",
+          "spec_url": "https://w3c.github.io/webappsec-trusted-types/dist/spec/#dom-trustedhtml-tojson",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "Currently Chrome Canary only"
```

Comment:
Very few things are enabled in Chrome Canary only in the same way that Firefox does, unless perhaps we count all of the "experimental web platform features" flag in this bucket.

Anyway, how did you determine this is in Canary only? Judging by https://mdn-bcd-collector.appspot.com/tests/api/TrustedHTML/toJSON it's already shipped to stable in Chrome 90, and that happened in this commit:
https://storage.googleapis.com/chromium-find-releases-static/0db.html#0dbce1408181f002902cba22fa275f658e8d9c18

Reply:
I trusted @RachelAndrew comment in  mdn/content#297, and it could have changed in the meantime!
Should we put 90?

Reply:
Yeah, I'd use Chrome 90 and then `npm run mirror` to update the rest.

Reply:
I tried to do it. My first run wiith `npm run mirror`, so I hope I did well. Fix in next commit

Reply:
I didn't try running it myself, but `npm run mirror edge` should have mirrored the Edge data, but you have to run it once for each file you care about so it's very tedious. https://github.com/mdn/browser-compat-data/pull/10645 will make it less tedious.

Reply:
I tried, but I think it mirrored it to false as it took IE as source. Maybe I did it wrong, though.

Reply:
Ah right, https://github.com/mdn/browser-compat-data/pull/6582 hasn't been merged yet :)